setup.py

```diff
@@ -33,12 +33,13 @@
         'License :: OSI Approved :: MIT License',
         'Natural Language :: English',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.6'
     ],
     install_requires=[
+        'future',
         'futures',
         'threadloop>=1,<2',
-        # we want thrift>=0.9.2.post1,<0.9.3, but we let the users pin to that
-        'thrift',
+        'thrift>=0.10.0',
```

Comment:
many apps at uber are using 0.9.2 or 0.93, pinning the dependency like this is going to break them. Can we just document that with python 3 people should use >=0.10?

I am also not sure how the generated files will work with thrift < 0.10.

Internally we use `tox` to run unit tests with several versions of dependencies, e.g. tornado. I would like to do that here with thrift